prow/github/types.go

```diff
@@ -1099,3 +1106,12 @@ type ProjectCard struct {
 	ContentType string `json:"content_type"`
 	ContentURL  string `json:"content_url"`
 }
+
+// HmacSecret contains a hmac token and its expiration time.
+type HmacSecret struct {
```

Comment:
I don't think this is really a public github type, as such - it probably belongs in `hmac.go`, and probably as non-public.